src/snapred/backend/recipe/algorithm/CalibrationMetricExtractionAlgorithm.py

```diff
@@ -80,7 +80,9 @@ def PyExec(self):
 
             sigmaAverage = np.average(sigmas)
             strainAverage = np.average(strains)
-            twoThetaAverage = pixelGroupingParameters[index].twoTheta
+
+            # Convert to degrees from radians (?)
+            twoThetaAverage = pixelGroupingParameters[index].twoTheta * (180.0 / np.pi)
```

Comment:
Why is this required?

Reply:
**The initial test code for this section had its input in _degrees_, which indicated that `CalibrationMetric` is "expecting" to be initialized that way.**  (`PixelGroupingParameters` is calculated in radians.)  I personally would prefer if the conversion to degrees happened in the GUI panel code, but that probably should wait for refactoring.  In general, _radians_ would be considered to be the "mathematical unit".

Reply:
Gotcha, I would like to create a meta type at some point to help handle unit dependent values.